app/helpers/and.js

```diff
@@ -0,0 +1,4 @@
+import Ember from 'ember';
+import { andHelper } from 'ember-truth-helpers/helpers/and';
+
+export default Ember.Helper.helper(andHelper);
```

Comment:
> Do you have any clue how we can test this guard?

Its mostly to make us feel better (I know we are silly), so I'm fine with the fact that the tests all pass (which the do without the guard).